src/graphql/mutations.ts

```diff
@@ -37,8 +37,8 @@ export interface SignInEmailInput {
 }
 
 export const MUTATION_SIGN_IN = gql`
-  mutation signInEmail($email: String! $password: String!) {
-    signInEmail(email: $email password: $password) {
+  mutation signInEmail($email: String!, $password: String!) {
+    signInEmail(email: $email, password: $password) {
```

Comment:
FYI commas are not necessary here.

Reply:
resolved [29d69a5](https://github.com/dooboolab/hackatalk-mobile/pull/115/commits/29d69a5c3e9b42ec291363e1fc0ef2f35fa2fb25)